docs/example-scenario/apps/internal-apim-ase-scenario.md

```diff
@@ -0,0 +1,193 @@
+
+### Scalability
+
+API Management instances could be [scaled out][apim-scale] depending upon a number of factors like number and rate of concurrent connections, the kind and number of configured policies, request and response sizes, and back-end latencies on the APIs. Scaling out instance options are available in Basic, Standard and Premium Tiers but are bound by an upper scale limit in tiers below premium. The instances are generally referred to as Units and can be scaled up to a max of 2 units in Basic tier, 4 units in Standard tier and any number of units in the Premium tier. [Auto Scaling][apim-autoscale] options are also available to enable scale out based on rules.
+
+App Service Environments are designed for scale with limits based on the pricing tier and the apps hosted under the App Service Environments can be [configured to scale out (number of instances) or scale up (instance size)][ase-scale] depending upon the requirements of the application. 
+
+Azure Application Gateway auto scaling is available as a part of the Zone redundant SKU in all public Azure regions. See the [public preview feature][appgtwy-scale] regarding App gateway Auto scaling.
+
+- For scalability topics, see the [scalability checklist][scalability] available in the Azure Architecture Center.
+
+### Security
+
+Since the above example scenario is hosted completely on an internal network, API Management and ASE are already deployed on [secured infrastructure (Azure VNET)][vnet-security]. Application Gateways can be [integrated with Azure Security Center][appgtwy-asc] to provide a seamless way to prevent, detect, and respond to threats to the environment.
+
+- For general guidance on designing secure solutions, see the [Azure Security Documentation][security]
+
+### Resiliency
+
+This example scenario though talks more about configuration, the APIs hosted on the App Service Environments should be resilient enough to handle errors in the requests which eventually is managed by the API Management service and Application Gateway. Consider [Retry and Circuit breaker patterns][api-pattern] in the API design. 
+- For general guidance on designing resilient solutions, see [Designing resilient applications for Azure][resiliency].
+
+## Deployment scenario
+
+### Prereqs and assumptions
+1. Custom Domain Name purchased.
+2. An SSL certificates (we used a wild card one from Azure Certificates Service) to use one for all our custom domains.
+3. This specific deployment uses the domain name contoso.org and a wild card SSL certificate for the domain.
+4. The deployment is using the resource names and address spaces mentioned in the deployment section which can be configured. 
+
+### Deployment and putting the pieces together  
+
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.svg)](https://deploy.azure.com/?repository=https://github.com/ssarwa/API-Management-ASE-AppGateway)
+
+The components deployed using the above ARM template
```

Comment:
If the instructions are in the Github repository, let's not restate them here but instead just link to it.

Reply:
The ARM template only creates a skeleton of the resources. There needs to be additional configuration done which is explained here.